migrations/staging/20171031131614_stg_wmt_extract_sa.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.withSchema('staging').createTable('wmt_extract_sa', function (table) {
+    table.increments('id')
+    table.string('case_ref_no')
+    table.string('tier_code')
+    table.string('team_code')
+    table.string('om_grade_code')
+    table.string('om_key')
+    table.string('location')
+    table.string('disposal_type_desc')
+    table.string('disposal_type_code')
+    table.string('standalone_order')
+  }).catch(function (error) {
+    console.log(error)
```

Comment:
Should we be using the logger, that way we will be able to audit it

Reply:
Probably. Good point. I'll update here and in other migrations files